wiki/Layout-component/Layout.md

```diff
@@ -39,4 +37,128 @@ The Layout core component helps us to layout our different components with a mor
+
+
+<br />[Back to top](#wiki-wrapper)<br /><br /><br />
+
+
+
+### 2. Children 
+
+`required` `container`
+
+The children property allows the adopter to place their content within the correct column to be laid out. The direct child `HTML`/`JSX` provided gets the column styling. For example if you choose a 2 column layout you will need 2 child wrappers, or 3 columns you will need 3 child wrappers. See the code examples below.
+
+###### Examples
+
+![layout](https://user-images.githubusercontent.com/3793636/122071996-d372e480-cdbc-11eb-8242-38eaee8d583f.gif)
+
+```html
+<!-- Here is an example where you’re using **two** columns. -->
+
+<Layout type=“4-12”>
+  <div>{content for column 1}</div>
+  <div>{content for column 2}</div>
+</Layout>
+
+<!-- Here is an example where you might have **four** columns. -->
+
+<Layout type=“4-4-4-4”>
+  <div>{content for column 1}</div>
+  <div>{content for column 2}</div>
+</Layout>
+
+<!-->
+    Note that `4-4-4-4` layout is not an option at this time.
+
+    Also, notice how each column has a corresponding child.
+    You could also optionally leave off a column from right
+    to left if you don’t need it for a given layout.
+ </-->
+```
+
+> ![image](https://user-images.githubusercontent.com/3793636/117873641-a6835d00-b265-11eb-8433-8c9c73a2e999.png) Note the code examples above maybe react specific, but the functionality should stand across any language with how it relates to being built out.
+
+<br />[Back to top](#wiki-wrapper)<br /><br /><br />
+
+
+
+### 3. Nested 
+
+`optional` `boolean`
+
+The nested option allows a given layout to sit comfortably within the [[Carbon 2x Grid]] allowing everything to continue to line up properly.
+
+###### Logic
+
+|     | condition          | description | |
+|:----|:-------------------|:------------|:-|
+| 3.1 | `nested === true`  | If nested is `true` do not include the `.bx—grid` so it can be nested within [[Carbon 2x Grid]]. Everything should still align properly. |  |
+| 3.2 | `nested === false` | If `false` render it as a normal [[Carbon 2x Grid]] including the `.bx—grid` | `default` |
+
+###### Example 
+
+![layout--nested](https://user-images.githubusercontent.com/3793636/122094807-2e630680-cdd2-11eb-898d-9bb8c9301b1d.gif)
+
+<br />[Back to top](#wiki-wrapper)<br /><br /><br />
+
+
+
+### 4. Border
+
+`optional` `boolean`
+
+The optional border is displayed at the bottom of the [[Layout]] providing a visual indicator for a change of topic.Typically used when nesting the component.
+
+###### Logic
+
+|     | condition          | description | |
+|:----|:-------------------|:------------|:-|
+| 3.1 | `border === true`  | If border is `true` then the pattern’s border at the bottom should render. |  |
```

Comment:
Do we want to reference "pattern" here? I'd guess this is carry over from when we called layout component's patterns?

```suggestion
| 3.1 | `border === true`  | If border is `true` then the layout component's border at the bottom should render. |  |
```

Reply:
ah, great catch. I've been trying to get all of these old pattern references.